README.md

```diff
@@ -187,7 +183,7 @@ which means you can modify it, redistribute it or use it however you like.
                                      of SIZE.
     --playlist-reverse               Download playlist videos in reverse order
     --xattr-set-filesize             Set file xattribute ytdl.filesize with
-                                     expected filesize (experimental)
+                                     expected file size (experimental)
```

Comment:
Autogenerated from where?
I was not able to find anything else than in the README and in options.py: https://github.com/rg3/youtube-dl/search?utf8=%E2%9C%93&q=--xattr-set-filesize&type=Code